api/room/schema.py

```diff
@@ -70,7 +70,7 @@ class Arguments:
         room_type = graphene.String()
         capacity = graphene.Int(required=True)
         image_url = graphene.String()
-        location_id = graphene.Int()
+        location_id = graphene.Int(required=True)
```

Comment:
Kindly remove the required=True. Some rooms in the db don't have this field, and at the moment it's not a must to include location field. This will updated once the new structure is in place.

Reply:
@bevkololi without the location, we'll have results that are not correct. This is to ensure that for every new room created, the location needs to be specified such that one can view the analytics of the location they belong to. This is how it was implemented with the REST endpoint.